airflow/jobs/scheduler_job.py

```diff
@@ -1006,8 +978,7 @@ def _change_state_for_tis_without_dagrun(self,
             )
             Stats.gauge('scheduler.tasks.without_dagrun', tis_changed)
 
-    @provide_session
-    def __get_concurrency_maps(self, states, session=None):
+    def __get_concurrency_maps(self, states, session):
```

Comment:
This method has an invalid rtype. Returns two dictionaries in a tuple, not just one.